apps/meteor/app/apps/server/converters/messages.js

```diff
@@ -14,6 +14,59 @@ export class AppMessagesConverter {
 		return this.convertMessage(msg);
 	}
 
+	async convertMessageRaw(msgObj) {
+		if (!msgObj) {
+			return undefined;
+		}
+
+		const { rid, editedBy, attachments, u, ...message } = msgObj;
+
+		const getRoom = () => ({ id: rid });
+
+		const getEditor = () => (editedBy ? { id: editedBy._id, username: editedBy.username } : undefined);
+
+		const getAttachments = async () => this._convertAttachmentsToApp(attachments);
```

Comment:
Whew, just checked and `_convertAttachmentsToApp` doesn't do any queries to the DB :relieved:

Reply:
Yes, it doesn't make any DB calls.